app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,221 @@
  *= require_tree .
  *= require_self
  */
+
+* {
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+.container {
+  max-width: 500px;
+  margin: 0 auto;
+  padding: 10px;
+}
+
```

Comment:
- [ ] Following the project requirements, you should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
- Colors
- Typography: font face, size and weight.
- Layout: composition and space between elements.